### DISCUSSION TITLE
How to map previous/next to keys?

### DISCUSSION THREAD

**Discussion by HubKing** [tomasklaen/uosc#580] [Q&A]
It seems that UOSC automatically adds files in the same directory, and I can navigate the list using the < > buttons. But how can I map those to keys? I have tried the following in `input.conf`, but nothing happened.

```
PGUP playlist-prev
PGDWN playlist-next
```

**Comment by tomasklaen** [comment]
How to bind commands is described at the beginning of [Commands documentation](https://github.com/tomasklaen/uosc#commands).

```
>   script-binding uosc/next
<   script-binding uosc/prev
```

**Comment by OmiChrome** [comment]
now it seems like there is going to so many question related to MPV , not UOSc :(
